octokit/meta.go

```diff
@@ -0,0 +1,104 @@
+package octokit
+
+import (
+	"net"
+	"strconv"
+
+	"github.com/jingweno/go-sawyer/hypermedia"
+)
+
+// https://developer.github.com/v3/meta/
+var (
+	MetaURL = Hyperlink("/meta")
+)
+
+// Meta return an APIInfo with the current API meta information
+func (c *Client) Meta(uri *Hyperlink) (info APIInfo, result *Result) {
```

Comment:
I was a little confused if I should accept a `*url.URL`, a `*Hyperlink`, or nothing at all here.